.github/workflows/integration_test_charm.yaml

```diff
@@ -38,6 +38,11 @@ on:
           Required if `cloud` input is "microk8s"
         required: false
         type: string
+      metallb-addon:
+        description: |
+          Whether to enable metallb
+          required: false
```

Comment:
besides maybe a bigger setup time, is there any problem in always enabling it?

Reply:
i wouldnt mind enabling it always, dont see a downside. but i'd weigh @carlcsaposs-canonical's opinion here as well

Reply:
No strong opinion here. We will need it for COS tests one day as well.
Not sure it worth to enable it always and wait-for-enabling in all unrelated tests.

Reply:
add default to `false`

Reply:
defaulted to `false` in [25b6845](https://github.com/canonical/data-platform-workflows/pull/244/commits/25b68454267ce858ac763fcd5bf617d0f98a3ff2)